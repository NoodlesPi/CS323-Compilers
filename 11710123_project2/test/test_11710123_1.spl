// type 1 and type 5
int main(){
    int a = 10;
    a = b;
    return 0;
}