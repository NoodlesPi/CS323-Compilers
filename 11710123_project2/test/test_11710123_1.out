Error type 1 at Line 4: variable is used without definition
Error type 5 at Line 4: unmatching types on both sides of assignment operator (=)
